### PR TITLE
adding supercomphud

### PR DESCRIPTION
**Pull request by lavaOnoes**

### CODE REVIEW

File: hud-data/supercomphud-omega.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "materials"
+  ],
+  "repo": "https://github.com/lavaOnoes/SUPERCOMPHUD-OMEGA",
+  "hash": "89dc5548d68b4b5426799f0702878e8e9855044b",
+  "prerelease": false,
+  "resources": [
+    "supercomphud-main",
+    "1",
+    "counter",
+    "https://www.youtube.com/watch?v=q8LgIzs_M2o"
```
Comment (mastercoms) [inline hud-data/supercomphud-omega.json:18]: `https://youtu.be/q8LgIzs_M2o` instead

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: hud-data/supercomphud-omega.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "SUPERCOMPHUD-OMEGA",
+  "author": "kachowhero, lava",
+  "releaseDate": "idk lol",
```
Comment (mastercoms) [inline hud-data/supercomphud-omega.json:4]: You can remove this, it's optional

File: hud-data/supercomphud-omega.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "SUPERCOMPHUD-OMEGA",
+  "author": "kachowhero, lava",
```
Comment (mastercoms) [inline hud-data/supercomphud-omega.json:3]: kachowhero should be listed as a contributor instead

**Comment by lavaOnoes** [comment]
is my submission denied

**Comment by mastercoms** [comment]
No sorry I just have been really busy, I'm sorry!

**Comment by lavaOnoes** [comment]
oh i see, soz soz

File: hud-data/supercomphud-omega.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    "materials"
+  ],
+  "repo": "https://github.com/lavaOnoes/SUPERCOMPHUD-OMEGA",
+  "hash": "89dc5548d68b4b5426799f0702878e8e9855044b",
+  "prerelease": false,
+  "resources": [
+    "supercomphud-main",
+    "1",
+    "counter",
+    "https://youtu.be/q8LgIzs_M2o"
```
Comment (rbjaxter) [inline hud-data/supercomphud-omega.json:18]: i'd remove this

Comment (lavaOnoes) [reply hud-data/supercomphud-omega.json:18]: may i ask why? aren't these for the images i have in hud-resources to be displayed? unless im doing it wrong

Comment (rbjaxter) [reply hud-data/supercomphud-omega.json:18]: i think the video you included is not the one you intended lmao

Comment (lavaOnoes) [reply hud-data/supercomphud-omega.json:18]: you'd be surprised

**Review by mastercoms — APPROVED** [review]

**Review by mastercoms — CHANGES_REQUESTED** [review]
Please update the supercomphud-main image to be cleaner/more appropriate.

**Review by mastercoms — APPROVED** [review]

**Pull request merged**